### PR TITLE
Add meeting.unset_logo  and meeting.unset_font action + tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: openslides_backend/action/meeting/unset_font.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    """
+    Action to unset a font form a meeting.
+    """
+
+    model = Meeting()
+    schema = DefaultSchema(Meeting()).get_default_schema(
+        title="meeting unset font schema",
+        description="An array of meeting_id and place to unset the font.",
+        additional_required_fields={
+            "meeting_id": {"type": "integer"},
```
Comment (jsangmeister) [inline openslides_backend/action/meeting/unset_font.py:20]: please use the schemas provided in `schema.py`, in this case `required_id_schema`

However, see https://github.com/OpenSlides/openslides-backend-coordination/issues/155#issuecomment-721823964: Maybe that's not necessary and we can just use the normal update schema.

File: openslides_backend/action/meeting/unset_font.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            "place": {"type": "string"},
```
Comment (jsangmeister) [inline openslides_backend/action/meeting/unset_font.py:21]: I think `minLength: 1` is missing, place cannot be empty

File: openslides_backend/action/meeting/unset_logo.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    """
+    Action to unset a logo form a meeting.
+    """
+
+    model = Meeting()
+    schema = DefaultSchema(Meeting()).get_default_schema(
+        title="meeting unset logo schema",
+        description="An array of meeting_id and place to unset the logo.",
+        additional_required_fields={
+            "meeting_id": {"type": "integer"},
```
Comment (jsangmeister) [inline openslides_backend/action/meeting/unset_logo.py:20]: see above

File: openslides_backend/action/meeting/unset_logo.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            "place": {"type": "string"},
```
Comment (jsangmeister) [inline openslides_backend/action/meeting/unset_logo.py:21]: see above

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/meeting/unset_font.py (hunk 1 above)
Comment (jsangmeister) [reply openslides_backend/action/meeting/unset_font.py:20]: Update: See https://github.com/OpenSlides/openslides-backend-coordination/issues/155

File: openslides_backend/action/meeting/unset_logo.py (hunk 4 above)
Comment (reiterl) [reply openslides_backend/action/meeting/unset_logo.py]: Updated.

File: openslides_backend/action/meeting/unset_logo.py (hunk 3 above)
Comment (reiterl) [reply openslides_backend/action/meeting/unset_logo.py]: Use update schema.

File: openslides_backend/action/meeting/unset_font.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/meeting/unset_font.py]: Used update schema, and updated payload.

**Comment by reiterl** [comment]
Reworked the unset actions, used the update schema and used the new payload, (id, place)

**Review by jsangmeister — APPROVED** [review]
Please rebase (for models checksum) and squash

**Comment by reiterl** [comment]
The tests of the new master failed:
FAILED tests/system/action/test_delete_cascade.py::TestDeleteCascade::test_cascade_overwrite_protect
FAILED tests/system/action/test_delete_cascade.py::TestDeleteCascade::test_cascade_protect

**Pull request merged**